### PR TITLE
zabbix-proxy role Debian arm64 support fix

### PR DESCRIPTION
**Pull request by gvol1222**
##### SUMMARY

Added checks for installing Debian family arm64 repositories to zabbix-proxy role

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

Adapted with zabbix-server role as guide

```paste below
repos installed before (in sources.list.d) (Ubuntu 22):

deb-src http://repo.zabbix.com/zabbix/6.2/ubuntu/ jammy main
deb http://repo.zabbix.com/zabbix/6.2/ubuntu/ jammy main

Not finding the correct architecture causes a fallback to zabbix-proxy 5.0.2 from ubuntu ports

after:

deb-src http://repo.zabbix.com/zabbix/6.2/ubuntu-arm64/ jammy main
deb http://repo.zabbix.com/zabbix/6.2/ubuntu-arm64/ jammy main

```

### CODE REVIEW

**Review by D3DeFi — APPROVED** [review]
Thank you for you contribution, this looks good to me, but I am not managing roles that much. 

Can we get one more review please? @mu1f407 @pyrodie18

File: roles/zabbix_proxy/tasks/Debian.yml
Comment (D3DeFi) [inline roles/zabbix_proxy/tasks/Debian.yml:141]:
```suggestion
- name: apt-get clean
  apt:
    clean: yes
    update_cache: yes
    cache_valid_time: 0
  become: true
  tags:
    - skip_ansible_lint
```

is this better?

File: roles/zabbix_proxy/tasks/Debian.yml
Comment (D3DeFi) [inline roles/zabbix_proxy/tasks/Debian.yml:230]: Wouldn't this also impact debian 11? Or is Deb9 setup similar to Deb11? (sorry I am absolutely out of the loop when it comes to this at the moment)

**Review by D3DeFi — APPROVED** [review]
I am fine with proposed changes. Not sure how long it will take till we get the second review here, so I will merge this, but can you first respond to my 2 comments here please?

File: roles/zabbix_proxy/tasks/Debian.yml
Comment (gvol1222) [reply roles/zabbix_proxy/tasks/Debian.yml:141]: Hello of course, just blindly copied server role here. Ansible bindings for apt do the trick in a cleaner way.

File: roles/zabbix_proxy/tasks/Debian.yml
Comment (gvol1222) [reply roles/zabbix_proxy/tasks/Debian.yml:230]: Again I followed the server role commands there. To be honest, meta-package default-mysql-client is present in all 3 supported Debian distros. I can't figure out why the mariadb-client is used instead at Deb10 below

Comment (pyrodie18) [reply roles/zabbix_proxy/tasks/Debian.yml:230]: Honestly not sure what this is going to do here and right now under the current 1.X branch we're not testing for most of the older OSs.

Comment (D3DeFi) [reply roles/zabbix_proxy/tasks/Debian.yml:230]: Yeah, ok. I wouldn't dissect this anymore.

For me this is OK to go. @pyrodie18 Feel free to merge if you have no further requests.

**Comment by D3DeFi** [comment]
Thank you for your contribution! 🙂

Been dragging the merge of this for too long. Sorry about that.

**Pull request merged**
